.env.example

```
@@ -1,5 +1,5 @@
 # Database
-DATABASE_URL="postgresql://user:password@localhost:5432/linkid"
+DATABASE_URL="postgresql://postgres:gpnashik8485@localhost:5432/linkid"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)